### ISSUE TITLE
Markdown preview nested lists shouldn't have bottom margin

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#124419]
This style fixes the issue:

```
ul ul {
    margin-bottom: 0;
}
```

<img width="707" alt="Screen Shot 2021-05-22 at 3 41 49 AM" src="https://user-images.githubusercontent.com/2193314/119223726-d9bcbc00-baaf-11eb-9b33-841d24cf36f6.png">
<img width="369" alt="Screen Shot 2021-05-22 at 3 42 24 AM" src="https://user-images.githubusercontent.com/2193314/119223728-dcb7ac80-baaf-11eb-8aea-cbd5ccacd977.png">

**Comment by Ashray123** [comment]
Hello @mjbvz I would love to take a shot at this, please guide me on what is needed to be done .

**Comment by inidaname** [comment]
@Tyriar should this also reflect with `ol` nested list use-case

```
ol ol {
   margin-bottom: 0;
}
```

**Comment by inidaname** [comment]
@mjbvz could a fix to this be added to #124385
